packages/extension/webpack.config.js

```diff
@@ -93,11 +93,12 @@ module.exports = {
     })
   ],
   resolve: {
-    alias: packages.reduce((alias, pkg) => {
-      alias[`@polkadot/${pkg}`] = path.resolve(__dirname, `../${pkg}/src`);
-
-      return alias;
-    }, {}),
+    alias: packages.reduce((alias, p) => ({
+      ...alias,
+      [`@polkadot/${p}`]: path.resolve(__dirname, `../${p}/src`)
+    }), {
+      'react/jsx-runtime': require.resolve('react/jsx-runtime')
```

Comment:
Next version of React will allow to get rid of this little hack - the commit was merged, should be in the next release. 

Tracked this from babel, to babel reverted, to react-as-logged-by-babel, to commit... 

However, came up with the suggested fix without having done that hunt, that only came afterwards, stumbled across the above links while seeing if there is a simpler/better way, however, the only suggestion I saw was for this alias version, as already done.